example/src/App.css

```diff
@@ -36,3 +36,9 @@
     transform: rotate(360deg);
   }
 }
+
+a[href^="http"]:not([data-internal-link]):before {
+  margin-left: 10px;
+  content: '[external]';
```

Comment:
I do not see an obvious way to include [this icon](https://materialui.co/icon/open-in-new) in css.

Reply:
What about not doing a CSS-only solution? Can we create a wrapper around `Link` that behaves differently based on the start of `href`?

https://mui.com/material-ui/react-link/

This is really bad and stupid and probably doesn't work. But maybe it conveys my thoughts?

```tsx
import * as React from 'react';
import Link as MuiLink from '@mui/material/Link';
import OpenInNewIcon from '@mui/icons-material/OpenInNew';

export default function Link(props) {
  if (!props.href.startsWith('http')) {
    return (<MuiLink {...props}/>);
  }
  return (
    <MuiLink
      {...props}
      target="_blank"
      rel="noopener"
    >
      <OpenInNewIcon/>{children}
    </MuiLink>
  );
}
```

I think https://github.com/remarkjs/react-markdown#appendix-b-components can be used with our new Markdown component to remap `<a>` tags to use this too.